Sources/ReactiveMoya/ReactiveSwiftMoyaProvider.swift

```diff
@@ -6,16 +6,36 @@ import Moya
 
 /// Subclass of MoyaProvider that returns SignalProducer instances when requests are made. Much better than using completion closures.
 open class ReactiveSwiftMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType {
-    private let stubScheduler: DateSchedulerProtocol?
+    private var stubScheduler: DateSchedulerProtocol?
     /// Initializes a reactive provider.
     public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
                 requestClosure: @escaping RequestClosure = MoyaProvider.defaultRequestMapping,
                 stubClosure: @escaping StubClosure = MoyaProvider.neverStub,
                 manager: Manager = ReactiveSwiftMoyaProvider<Target>.defaultAlamofireManager(),
-                plugins: [PluginType] = [], stubScheduler: DateSchedulerProtocol? = nil,
+                plugins: [PluginType] = [],
+                stubScheduler: DateSchedulerProtocol? = nil,
```

Comment:
This one parameter would have been without default value. Otherwise we have similar declarations, because if you don't need to specify this parameter, you have 2 identical initializers in most of the cases. Thus:
```swift
stubScheduler: DateSchedulerProtocol? = nil
```
-> 
```swift
stubScheduler: DateSchedulerProtocol?
```